test/CMakeLists.txt

```diff
@@ -1,3 +1,7 @@
+# GoogleTest builds with /MT(d) by default, which is incompatible with Valhalla's /MD(d) builds.
```

Comment:
could you leave just 1-2 words each what `/MT(D)` is?

Reply:
Sure thing!